src/ferc1_eia_match/config.py

```diff
@@ -0,0 +1,39 @@
+"""Helper functions to compute and output various metrics."""
+
+from pydantic import BaseModel
+
+
+class Inputs(BaseModel):
+    """Configuration for input resources."""
+
+    start_year: int
+    end_year: int
+
+
+class ColumnEmbedding(BaseModel):
+    """Specify embedding vectorizer to use for column and any options."""
+
+    embedding_type: str
+    options: dict = {}
```

Comment:
I'm interpreting `options` as potential keyword arguments to the embedding function specified by `embedding_type`? I'm going to just use the `config.embedding.embedding_map` as the input to the DataFrameEmbedder.

Reply:
Yeah that was the plan. And that makes sense, should be cleaner that way